rednotebook/help.py

```diff
@@ -215,6 +215,10 @@
 lists. If you want to hide words with special characters, you can
 escape them with a backslash: 3\\.50\\?
 
+You can limit number of displayed words by tweaking 'Tag display limit'
```

Comment:
*the* number of
*the* 'Tag display limit'

Reply:
Thanks :) - done!